pkg/diagnostics/cluster/registry.go

```diff
@@ -221,8 +233,15 @@ func (d *ClusterRegistry) checkRegistryLogs(pod *kapi.Pod, r types.DiagnosticRes
 	}
 	defer readCloser.Close()
 
+	// Indicator that selinux is blocking the registry from writing to disk:
+	selinuxErrorRegex, _ := regexp.Compile(".*level=error.*mkdir.*permission denied.*")
```

Comment:
This is the same message you get if your volume doesn't allow writing by any user, right?  If so, update the help message to note both problems.